pkg/virt-controller/watch/migration.go

```diff
@@ -1427,6 +1434,46 @@ func (c *MigrationController) updatePod(old, cur interface{}) {
 	return
 }
 
+// When a resourceQuota is updated, figure out if there are pending migration in the namespace
+// if there are we should push them into the queue to accelerate the target creation process
+func (c *MigrationController) updateResourceQuota(_, cur interface{}) {
+	curResourceQuota := cur.(*k8sv1.ResourceQuota)
+	log.Log.V(4).Object(curResourceQuota).Infof("ResourceQuota updated")
+	objs, _ := c.migrationInformer.GetIndexer().ByIndex(cache.NamespaceIndex, curResourceQuota.Namespace)
+	for _, obj := range objs {
```

Comment:
These callbacks should not block for a significant time. Consider wrapping this in a goroutine.

Reply:
IMHO this is unnecessary complexity.
@vladikr  WDTY?